README.md

```diff
@@ -39,9 +39,15 @@ If you need help with CAPIC, please visit the [#cluster-api-ionoscloud][slack] c
 
 ## Compatibility
 
-### Cluster API Versions (TODO)
+### Cluster API Versions
 
-### Kubernetes Versions (TODO)
+This provider's versions are compatible with the following versions of Cluster API:
+
+|                        | Cluster API v1beta1 (v1.7) |
+|------------------------|:--------------------------:|
+| CAPIC v1alpha1 (v0.2)  |             ✓              |
```

Comment:
this should also include CAPI v1.6

Reply:
As we haven't tested this with 1.6 - I would not include this at least not in this release. We can maybe run another test with 1.6 and add it to the matrix without a release

Reply:
As you like, 
But i just wanted the comparability table to look better with more support versions.

I know CAPI always works with +1/-1 minor version.